notebooks/metatranscriptomics_pipeline_iSeqnorm.ipynb

```diff
@@ -0,0 +1,1477 @@
+    "\n",
+    "##Can also import counts from Qiita per_sample_FASTQ summaries.  \n",
+    "# per_sample_fastq_counts_df = pd.read_csv('./test_data/Demux/YYYY_MM_DD_Celeste_Adaptation_16_17_18_21_per_sample_fastq.tsv',\n",
+    "#                                          sep='\\t')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Merge read_counts_df with plate_df \n",
+    "\n",
+    "plate_df_w_reads = merge_read_counts(plate_df,read_counts_df, reads_column_name='Raw Reads')\n",
+    "\n",
+    "plate_df_w_reads = merge_read_counts(plate_df_w_reads,read_counts_df,\n",
+    "                                     reads_column_name='Filtered Reads')\n",
+    "\n",
+    "# plate_df_w_reads = merge_read_counts(plate_df_w_reads,per_sample_fastq_counts_fp,\n",
+    "#                                      reads_column_name='Qiita Reads')\n",
+    "\n",
+    "plate_df_w_reads.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "reads_column = 'Raw Reads'\n",
+    "\n",
+    "f, ((ax1, ax2), (ax3, ax4)) = plt.subplots(nrows=2, ncols=2, figsize=(8, 8))\n",
+    "# evenness plot\n",
+    "rmax = int(round(plate_df_w_reads[reads_column].max(),-2))\n",
+    "survival_df = pd.concat([read_survival(plate_df_w_reads.loc[plate_df_w_reads['Blank'] == True,\n",
+    "                                                            reads_column], label='Blanks',rmax=rmax),\n",
+    "                         read_survival(plate_df_w_reads.loc[plate_df_w_reads['Blank'] == False,\n",
+    "                                                            reads_column], label='Samples',rmax=rmax)])\n",
+    "\n",
+    "ax3.set_xlabel(reads_column)\n",
+    "ax3.set_ylabel('Samples')\n",
+    "survival_df.plot(color = ['coral','steelblue'],ax=ax1)\n",
+    "ax1.set_xlabel(reads_column)\n",
+    "ax1.set_ylabel('Samples')\n",
+    "\n",
+    "##Histogram\n",
+    "sns.histplot(plate_df_w_reads[reads_column],ax=ax3)\n",
+    "\n",
+    "##Regressopm\n",
+    "sns.regplot(x=\"MiniPico Library DNA Concentration\", y=reads_column, data=plate_df_w_reads, ax = ax2);\n",
+    "\n",
+    "#Boxplot\n",
+    "sns.boxplot(x=\"Blank\", y=reads_column, data=plate_df_w_reads, ax = ax4);\n",
+    "sns.stripplot(x=\"Blank\", y=reads_column, data=plate_df_w_reads, ax = ax4,\n",
+    "              size=3,color='black',alpha=0.5)\n",
+    "\n",
+    "\n",
+    "plt.tight_layout()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 2: Calculate iSeqnorm pooling volumes"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plate_df_normalized = calculate_iseqnorm_pooling_volumes(plate_df_w_reads,dynamic_range=20,normalization_column='Raw Reads')"
```

Comment:
It would be useful to have a comment describing the source of the dynamic_range=20 hard-coding, and info on whether the user should ever be changing this value.

Reply:
this is actually a parameter that the technician has to play with until they find the optimal value. I will add an explanation.